src/test/css/completion.test.ts

```diff
@@ -209,6 +209,7 @@ suite('CSS - Completion', () => {
 				{ label: 'transition', resultText: 'body { transition:  ' }
 			]
 		});
+		// TODO: how to test it?
```

Comment:
@octref Can you help to create tests for this new setting? I tried and it was unsuccessful...

Reply:
- Make `testCompletionFor` take your completion setting
- In `ItemDescription` add command
- Assert that when you have the setting, retrigger command does not exist on the CompleionItem